docs/build-service.adoc

```diff
@@ -0,0 +1,10 @@
+== Build service
+
+To build service you can run:
+[source]
+----
+./msat-set-service-repo-url-metadata.sh application_name service_name organization_name
+----
+
+It will create service if it does not exist and set github repository as metadata property.
+To run this script environment variables MSAT_API_ENDPOINT, MSAT_USER_LOGIN and MSAT_USER_PASSWORD must be set.
```

Comment:
The MSAT API does not have a 'user login` and `user password`.

It has an `Access Key ID` and `Secret Access Key`